src/components/Button/Button.styles.ts

```diff
@@ -26,25 +26,23 @@ const StyledButton = styled.button<{
   padding: ${(props) =>
     props.isLoading
       ? `${props.noPadding ? "0" : props.theme.spacing.xs}`
-      : `${props.noPadding ? "0" : props.theme.spacing.xxs} ${
-          props.noPadding ? "0" : props.theme.spacing.xs
-        }`};
+      : `${
+          props.noPadding ? "0" : props.size === "lg" ? "5px" : props.theme.spacing.xxs
```

Comment:
Question; any reason this doesn't align with a spacing prop?

Reply:
yes for the 5px, according the ux there is no token at the moment and we can hardcode for now :)